src/dipdup/models.py

```diff
@@ -152,7 +154,10 @@ def get_merged_storage(self, storage_type: Type[StorageType]) -> StorageType:
 
         _logger.debug('After: %s', storage)
 
-        return storage_type.parse_obj(storage)
+        try:
+            return storage_type.parse_obj(storage)
+        except ValidationError:
```

Comment:
`except Error as e raise Error from e` as first exception is a direct cause of second one.